tests/www/test_views.py

```diff
@@ -270,6 +270,73 @@ def test_prefill_form_null_extra(self):
         cmv.prefill_form(form=mock_form, pk=1)
 
 
+class TestMultiDuplicateConnectionModelView(TestBase):
+    def setUp(self):
+        super().setUp()
+        conn1 = Connection(
+            conn_id='google_cloud_default',
+            conn_type='Google Cloud',
+            description='Default Google Cloud Connection',
+        )
+
+        # Already copied and modified
+        conn2 = Connection(
+            conn_id='mongodb',
+            conn_type='FTP',
+            description='MongoDB2',
+            host='localhost',
+            schema='airflow',
+            port='5567',
+        )
+
+        conn3 = Connection(
+            conn_id='mysql_connection',
+            conn_type='FTP',
+            description='MongoDB2',
+            host='localhost',
+            schema='airflow',
+            port='3306',
+        )
+
+        conn4 = Connection(
+            conn_id='postgres_connection_Copy(1)',
+            conn_type='FTP',
+            description='Postgres',
+            host='localhost',
+            schema='airflow',
+            port='3306',
+        )
+
+        self.clear_table(Connection)
+        self.session.add_all([conn1, conn2, conn3, conn4])
+        self.session.commit()
+
+    def tearDown(self):
+        self.clear_table(Connection)
+        super().tearDown()
+
+    def test_duplicate_connection(self):
+        """ Test Duplicate multiple connection with suffix"""
+
+        mock_form = mock.Mock()
+        mock_form.data = {"action": "mulduplicate", "rowid": [1, 2, 4]}
+        resp = self.client.post('/connection/action_post', data=mock_form.data, follow_redirects=True)
+
+        expected_result = {
+            'google_cloud_default',
+            'google_cloud_default_Copy(1)',
+            'mongodb',
+            'mongodb_Copy(1)',
+            'mysql_connection',
+            'postgres_connection_Copy(1)',
+            'postgres_connection_Copy(2)',
+        }
+        response = {conn[0].strip() for conn in self.session.query(Connection.conn_id).all()}
```

Comment:
```suggestion
        response = {conn[0] for conn in self.session.query(Connection.conn_id).all()}
```

strip shouldn't be needed.